server/src/main/java/com/objectcomputing/checkins/services/agenda_item/AgendaItem.java

```diff
@@ -0,0 +1,117 @@
+package com.objectcomputing.checkins.services.agenda_item;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.micronaut.data.annotation.AutoPopulated;
+import io.micronaut.data.annotation.TypeDef;
+import io.micronaut.data.model.DataType;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.annotation.Nullable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+@Table(name = "agenda_items")
+public class AgendaItem {
+
+    @Id
+    @Column(name = "id")
+    @AutoPopulated
+    @TypeDef(type = DataType.STRING)
+    @Schema(description = "id of this agenda item", required = true)
+    private UUID id;
+
+    @NotNull
+    @Column(name = "checkinid")
+    @TypeDef(type = DataType.STRING)
+    @Schema(description = "id of the checkin this entry is associated with", required = true)
+    private UUID checkinid;
+
+    @NotNull
+    @Column(name = "createdbyid")
+    @TypeDef(type = DataType.STRING)
+    @Schema(description = "id of the member this entry is associated with", required = true)
+    private UUID createdbyid;
+
+    @Nullable
+    @Column(name = "description")
+    @Schema(description = "description of the agenda item")
+    private String description;
+
+    public AgendaItem(@JsonProperty("checkinid") @TypeDef(type = DataType.STRING) @Valid @NotNull UUID checkinid,
```

Comment:
I had these annotations in my original code, but I don't think they are needed, I have since removed them. Can you remove these, run your tests and see if it still works.

Reply:
Removed @Schema annotation, and the tests still work.